manifests/server/conf.pp

```diff
@@ -161,10 +165,10 @@
     include '::haveged'
   }
 
+  $rhel_greater_then_6 = ( $facts['os']['name'] in ['RedHat','CentOS','Fedora'] ) and ( $facts['os']['release']['major'] > '6' )
```

Comment:
* It will be fine to remove the Fedora logic―if it's not in the `metadata.json`, it isn't supported by SIMP, the PMT, or `on_supported_os`.
* I'd prefer to test something like `$facts['os']['family'] == 'RedHat'` instead of `['os']['name']` so it will be easier to add support for other EL distros (like Oracle/[SIMP-3620](https://simp-project.atlassian.net/browse/SIMP-3620)).